crates/types/src/receipt/deploy_template.rs

```diff
@@ -55,7 +58,12 @@ impl TemplateReceipt {
         }
     }
 
-    /// Returns the deployed template address. Panics if deploy has failed.
+    /// Returns the deployed template address.
```

Comment:
Returns the deployed [`Template`] [`Address`]

Reply:
`Address` is not a valid type so the link wouldn't resolve. I'm still rewording this for more clarity :+1:

Reply:
My bad